internal/pkg/args/args.go

```diff
@@ -30,6 +30,13 @@ func SingleArg(argName string, validate func(value string) error) cobra.Position
 				Count:    len(args),
 			}
 		}
+		if args[0] == "" {
+			return &errors.SingleArgExpectedError{
+				Cmd:      cmd,
+				Expected: argName,
+				Count:    len(args),
+			}
+		}
```

Comment:
Probably could be merged into the condition above with `||`:

```go
if len(args) != 1 || args[0] == "" {
    ...
```

Reply:
Done, added empty arg test case as well